extension/src/test/suite/experiments/workspace.test.ts

```diff
@@ -534,11 +545,17 @@ suite('Workspace Experiments Test Suite', () => {
           },
           {
             description: '[exp-f13bca]',
+            detail: `Created:${formatDate(
+              '2020-12-29T15:26:36'
+            )}, loss:-, accuracy:-`,
             label: 'f0f9186',
             value: { id: 'exp-f13bca', name: 'exp-f13bca' }
           },
           {
             description: undefined,
+            detail: `Created:${formatDate(
+              '2020-12-29T15:25:27'
+            )}, loss:-, accuracy:-`,
```

Comment:
[Q] why don't there have a loss or accuracy value?

Reply:
This experiment (`55d492c9c633912685351b32df91bfe1f9ecefb9`) is a failed experiment and [has no `metrics`](https://github.com/iterative/vscode-dvc/blob/121f651589f2325cd1cef20ab7c93671e55d8322/extension/src/test/fixtures/expShow/base/output.ts#L1752).